Sources/Tokenizers/PostProcessor.swift

```diff
@@ -22,20 +22,30 @@ extension PostProcessor {
 }
 
 enum PostProcessorType: String {
-    case TemplateProcessing
+    case Template
     case ByteLevel
-    case RobertaProcessing
+    case Bert
+    case Roberta
+
+    static let BertProcessing = "Bert"
+    static let RobertaProcessing = "Roberta"
+    static let TemplateProcessing = "Template"
```

Comment:
I'm not sure this is worth doing, I find the aliases and the suffix removal distracting for little benefit. My original approach was to simply use the same names that appear in the json, so someone reading both could easily match.

Reply:
It's very hard to use JSON as the ground truth, as they cone in all shapes and sizes. 

Shorter name was needed for our models to work, but I've added the static variables for backward-compatibility, to avoid breaking the library for other users.

Reply:
I'd say the json names used in transformers should be quite stable right now. Why do your models require shorter names? (just curious)

Reply:
I guess no model "requires" specific names, and JSONs can always be changed, but it generally results in a snowballing set of changes that have to be applied on every platform...

In UForm I have identical tests that run for the same models on the same data across all 3 languages across ONNX, PyTorch, and CoreML. You can [check them here](https://github.com/ashvardanian/uform/blob/main/CONTRIBUTING.md).

If a certain behavior is standard in the more popular ports of the library (Python and JS), I assume Hugging Face may want to provide the same behavior here to encourage adoption. A lot of people would probably appreciate the portability 🤗